### ISSUE TITLE
#-1523249009------------- Links must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1726]
<strong>PageName:</strong> LayersPage  

<strong>Source:</strong>`<a id="edit-button-d5922a2daf70448183862962554137a6" href="/layers/d5922a2daf70448183862962554137a6/edit" aria-describedby="calcite-tooltip-409662af-6b2e-da09-c63a-2b98e4d0c58b">`

<strong>Element Location: </strong>#edit-button-d5922a2daf70448183862962554137a6

<strong>Impact: </strong>serious

<strong>Failure Summary:</strong> Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

<strong>Tags:</strong> cat.name-role-value,wcag2a,wcag412,wcag244,section508,section508.22.a,ACT

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/link-name?application=axeAPI
